Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -478,7 +478,8 @@ def SwapKeepAndDrop(l):
         'keep Phase2TrackerDigiedmDetSetVector_mix_*_*',
         'keep *_TTClustersFromPhase2TrackerDigis_*_*',
         'keep *_TTStubsFromPhase2TrackerDigis_*_*',
-        'keep *_TrackerDTC_*_*'])
+        'keep *_TrackerDTC_*_*',
+        'keep *_*_Level1TTTracks_*'])
```

Comment:
Are you sure you need to add `*_*_Level1TTTracks_*`? I think the L1 tracks are already included in the FEVTEventContent (they are used for the upcoming re-reco of the HLT TDR samples).